### PR TITLE
Added android ios release blog post

### PR DESCRIPTION
**Pull request by stevengill**

### CODE REVIEW

File: www/_posts/2016-08-22-android-ios-release.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+To add it explicitly:
+
+    cordova platform add android@5.2.2
+    cordova platform add ios@4.2.1
+
+<!--more-->
+## What's new in Android
+
+* [CB-11615](https://issues.apache.org/jira/browse/CB-11615) updated `cordoova-common` to `1.4.0`
```
Comment (dpogue) [inline www/_posts/2016-08-22-android-ios-release.md]: typo: cordova-common

Comment (stevengill) [inline www/_posts/2016-08-22-android-ios-release.md]: good catch

**Comment by stevengill** [comment]
I'm actually going to publish updated docs and point to https://github.com/apache/cordova-docs/commit/af2ede15180d202e59d576f8f44e967018a268d1 for the edit-config stuff

**Comment by shazron** [comment]
For ios, we need to add the note like in cordova-ios 4.2.0:
https://cordova.apache.org/announcements/2016/06/23/ios-4.2.0.html

**Comment by stevengill** [comment]
This has been merged

**Pull request closed without merging**